app/database/db.go

```diff
@@ -411,6 +411,23 @@ func (conn *DB) Exec(sqlQuery string, values ...interface{}) *DB {
 	return newDB(conn.db.Exec(sqlQuery, values...))
 }
 
+var nowExpr = gorm.Expr("NOW()")
```

Comment:
The `db.go` file is already quite long, shouldn't you move it to something like `db_now.go` or `db_functions.go`?

Reply:
fixed